x/group/keeper/keeper_test.go

```diff
@@ -2321,6 +2321,198 @@ func (s *TestSuite) TestExecProposal() {
 	}
 }
 
+func (s *TestSuite) TestLeaveGroup() {
```

Comment:
Can we add on test, in TestVote, that does:
- on a proposal, some people vote
- one member leaves the group
- the expected tally result should NOT take into account the member who left